kernel-install/50-mkosi.install

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1+
+
+import argparse
+import logging
+import os
+import shutil
+from pathlib import Path
+from typing import NamedTuple, Optional
+
+from mkosi.config import OutputFormat, __version__
+from mkosi.log import die, log_setup
+from mkosi.run import run, uncaught_exception_handler
+from mkosi.types import PathString
+
+
+class Context(NamedTuple):
+    command: str
+    kernel_version: str
+    entry_dir: Path
+    kernel_image: Path
+    initrds: list[Path]
+    staging_area: Path
+    layout: str
+    image_type: str
+    initrd_generator: Optional[str]
+    uki_generator: Optional[str]
+    verbose: bool
+
+
+def we_are_wanted(context: Context) -> bool:
+    return context.uki_generator == "mkosi" or context.initrd_generator in ("mkosi", "mkosi-initrd")
+
+
+def mandatory_variable(name: str) -> str:
+    try:
+        return str(os.environ[name])
```

Comment:
`os.envget` can return `None`, but `os.environ` is str-only, no? mypy doesn't complain about this for me.
```suggestion
        return os.environ[name]
```